docs/knowledgebase/runtime/metadata.md

```diff
@@ -456,8 +456,6 @@ the FRAME System pallet is declaring that it may raise the
 
 <!--
 ### Examples
-
--
```

Comment:
@riusricardo So having blank lines inside HTML comments breaks the page?

Reply:
Not sure if it is the case for everything. I found out that it happens with the HTML comments. It breaks the TOC on the right.